### PR TITLE
StaticLLMPipeline: Align MAX_PROMPT_LEN

### PR DESCRIPTION
**Pull request by TolyaTalamanov**

### CODE REVIEW

File: src/cpp/src/llm_pipeline_static.cpp [hunk 1]
```diff
@@ -404,7 +428,7 @@ void StaticLLMPipeline::setupAndCompileModels(
     m_prefill_model = m_kvcache_model->clone();
     m_prefill_model->set_friendly_name(m_kvcache_model->get_friendly_name() + "_prefill");
     // (7) Reshape both models to static shape
-    const uint32_t kMaxPromptLen = pop_int_and_cast(properties, "MAX_PROMPT_LEN").value_or(1024u);
+    const uint32_t kMaxPromptLen = align_to(pop_int_and_cast(properties, "MAX_PROMPT_LEN").value_or(1024u), 64u);
     const uint32_t kMinResponseLen = pop_int_and_cast(properties, "MIN_RESPONSE_LEN").value_or(128u);
```
Comment (dmatveev) [inline src/cpp/src/llm_pipeline_static.cpp:432]: as in your picture the context size is defined by `kMaxPromptLen +  kMinResponseLen`, I think you need to align your  `kMinResponseLen` too.

Comment (TolyaTalamanov) [reply src/cpp/src/llm_pipeline_static.cpp]: Done

File: src/cpp/src/llm_pipeline_static.cpp [hunk 2]
```diff
@@ -404,8 +428,8 @@ void StaticLLMPipeline::setupAndCompileModels(
     m_prefill_model = m_kvcache_model->clone();
     m_prefill_model->set_friendly_name(m_kvcache_model->get_friendly_name() + "_prefill");
     // (7) Reshape both models to static shape
-    const uint32_t kMaxPromptLen = pop_int_and_cast(properties, "MAX_PROMPT_LEN").value_or(1024u);
-    const uint32_t kMinResponseLen = pop_int_and_cast(properties, "MIN_RESPONSE_LEN").value_or(128u);
+    const uint32_t kMaxPromptLen = align_to(pop_int_and_cast(properties, "MAX_PROMPT_LEN").value_or(1024u), 64u);
+    const uint32_t kMinResponseLen = align_to(pop_int_and_cast(properties, "MIN_RESPONSE_LEN").value_or(128u), 64u);;
```
Comment (dmatveev) [inline src/cpp/src/llm_pipeline_static.cpp:432]: `;;` but if clang-format doesn't complain then y not

Comment (TolyaTalamanov) [reply src/cpp/src/llm_pipeline_static.cpp]: Done

**Review by andrei-kochin — APPROVED** [review]

**Review by dmatveev — APPROVED** [review]

**Pull request merged**